packages/components/src/components/label/label.scss

```diff
@@ -24,6 +24,21 @@
             border-radius: $BORDER_RADIUS * 3;
         }
     }
+    &--adjustment {
+        color: var(--status-adjustment);
+
+        &:after {
```

Comment:
the &:after styling is the same for every `--modifier` except for the `background-color` is there a way to share this styling? 🤔